src/main/scala/io/sdkman/repos/VersionsRepo.scala

```diff
@@ -96,5 +96,7 @@ case class Version(
     platform: String,
     url: String,
     vendor: Option[String] = None,
-    visible: Option[Boolean] = Some(true)
+    visible: Option[Boolean] = Some(true),
+    algorithm: Option[String] = None,
+    checksum: Option[String] = None
```

Comment:
I was thinking along the lines of something like:

```suggestion
    checksums: Map[String, String] = emptyMap()
```